src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/IExceptionInstance.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.Diagnostics.Monitoring.WebApi.Exceptions
 {
     internal interface IExceptionInstance
     {
+        ulong Id { get; }
```

Comment:
Is `InnerExceptionIds` intentionally omitted from here?

Reply:
No, forgot it because it's not being used yet in the product.

Reply:
Tests now use the exception instance interface, so this shouldn't happen again.